### PR TITLE
Update ebsco-eds gem.

### LINKED ISSUE #1819: Update latest EDS gem
This should resolve https://github.com/sul-dlss/SearchWorks/issues/1800 and related, as well as current pagination issues.  

The flag to apply case tense for language and geography facets should be turned OFF - that is we want no change to the way those facets are displayed at this time.

### PR DESCRIPTION
**Pull request by jkeck**
Closes #1819

### CODE REVIEW

File: app/models/eds/session.rb [hunk 1]
```diff
@@ -19,20 +19,21 @@ def object
...
+        profile:                      Settings.EDS_PROFILE,
+        use_cache:                    Settings.EDS_CACHE,
+        eds_cache_dir:                Settings.EDS_CACHE_DIR,
+        timeout:                      Settings.EDS_TIMEOUT,
+        open_timeout:                 Settings.EDS_OPEN_TIMEOUT,
+        api_hosts_list:               Settings.EDS_HOSTS,
+        debug:                        Settings.EDS_DEBUG,
+        log:                          File.join(Settings.EDS_LOGDIR, 'faraday.log'),
+        decode_sanitize_html:         true,
+        recover_from_bad_source_type: true
```
Comment (jkeck) [inline app/models/eds/session.rb]: This line being added is the only non-formatting change to this file.

**Comment by jkeck** [comment]
AFAICT 0.3.18.pre (as tagged in GH) is the same as 0.3.19.pre.  When 0.3.18.pre was released it didn't have the appropriate changes so the gem developer re-released (w/o any change other than the version bump).

**Comment by jkeck** [comment]
@drh-stanford I should also mention this is current deployed to stage.

**Comment by drh-stanford** [comment]
Gotcha 👍

**Pull request merged**